src/builder.js

```diff
@@ -154,11 +154,11 @@ builder.build = function(project, uuid, path, gitBranch, branch, dockerOptions)
     buildLogger.info('[%s] Created tarball for %s', buildId, uuid);
 
     return docker.buildImage(project, tarPath, imageId + ':' + branch, buildId, buildLogger, dockerOptions, uuid);
-  }).then(function() {
-    buildLogger.info('[%s] %s built succesfully', buildId, uuid);
-    buildLogger.info('[%s] Tagging %s', buildId, uuid);
+  }).then(function(realBuildId) {
+    buildLogger.info('[%s] %s built succesfully as imageId: %s', buildId, uuid, realBuildId);
```

Comment:
yes, the `buildId` actually comes as e.g `namshi/roger:1.4.1` which is the buildName (or tagName) while the `realBuildId` comes as the ID of the buildName. Didn't want to change much in the variable names to avoid unforeseeable issues.